### PR TITLE
javascript-data-model - Sina Daryaei 

### PR DESCRIPTION
**Pull request by sinadaryaei**
image

### CODE REVIEW

File: javascript-data-model/order-history.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+var orderHistory = [
+  {
+    Order: 'August 4, 2020',
+    Total: '34.00',
+    Shiping: 'Js Masher',
+    OrderNumber: '114-3941689-8772232',
+    Item: 'Javascript for impatient programmers'
```
Comment (shawnkost) [inline javascript-data-model/order-history.js:7]: Property names should always start with lowercase! Property names are just like variables in which they should be using camelCase.

**Review by shawnkost — CHANGES_REQUESTED** [review]
Hey Sina! One change requested here!

File: javascript-data-model/order-history.js (hunk 1 above)
Comment (sinadaryaei) [reply javascript-data-model/order-history.js:7]: Making changes now.

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**